### DISCUSSION TITLE
How do I get both Hand and Pose and Face to be transfer.

### DISCUSSION THREAD

**Discussion by mbxymail** [cgtinker/BlendArMocap#149] [General]
I have a movie with motion and when I do a pose load and then a hands load the hands are lock to one location.

Is the a sequence to Load pose, face and hand at one time? THanks Help

https://user-images.githubusercontent.com/22260798/230511557-46159035-8a5f-4ede-8bdc-6abf23667ea7.mp4

![hands](https://user-images.githubusercontent.com/22260798/230511742-f746a11a-b090-4ccd-88aa-0a7c56ff4e67.png)

**Comment by cgtinker** [comment]
Use holistic as target?
